lib/modules/manager/gradle-wrapper/artifacts.ts

```diff
@@ -84,6 +87,8 @@ export async function updateArtifacts({
     } else {
       cmd += ` --gradle-version ${quote(config.newValue!)}`;
     }
+    cmd += ` --project-dir ${localGradleDir}`;
```

Comment:
i think you should instead pass one of the cwd exec options like all other managers do.